frontend/src/app/general/nav-bar-menu/nav-bar-menu.component.html

```diff
@@ -5,13 +5,14 @@
 
 <mat-list>
   <div *ngFor="let item of navBarService.navBarItems">
-    <div [routerLink]="item.routerLink">
+    <div>
```

Comment:
We should be able to remove this intermedia div.

Reply:
Ah right, think I got rid of any empty tags this time